cip/1.accepted/CIP2017-02-07-Map-Projection.adoc

```diff
@@ -0,0 +1,121 @@
+= CIP2017-02-07 Map Projection
+:numbered:
+:toc:
+:toc-placement: macro
+:source-highlighter: codemirror
+
+*Author:* Tobias Lindaaker <tobias.lindaaker@neotechnology.com>
+
+toc::[]
+
+== Map Projection
+
+Map projection takes a map or entity (node or relationship) and creates a map based on selected properties from map or entity.
+The map projection largely reuses the map syntax of curly braces and comma separated entries.
+But the projection entries are just a single identifier starting with a dot (.).
+The projected map entries use the same key in the projected map as in the source map.
+
+The following example:
+
+[source, cypher]
+----
+someMap {.alpha, .beta}
+----
+
+is thus equivalent to:
+
+[source, cypher]
+----
+{alpha: someMap.alpha, beta: someMap.beta}
+----
+
+In order to be further useful, map projection allow other types of selectors to augment the projected map.
```

Comment:
It would be useful with an example of, or some more discussion around, these other types of selectors.